ann_benchmarks/results.py

```diff
@@ -84,7 +84,7 @@ def load_all_results(dataset: Optional[str] = None,
     Yields:
         tuple: A tuple containing properties as a dictionary and an h5py file object.
     """
-    for root, _, files in os.walk(build_result_filepath(dataset, count)):
+    for root, _, files in os.walk(build_result_filepath(dataset, count, batch_mode=batch_mode)):
```

Comment:
If `data_export.py` is expected to export all the results that are found, then here `build_result_filepath` should return all the sub-directories.

However, in `def build_result_filepath(...):`

https://github.com/erikbern/ann-benchmarks/blob/3982de0c267c9e3de65044b5c55f143917df7023/ann_benchmarks/results.py#L11-L38

If `count` and `batch_mode` are `None`, directories of specific counts and batch mode will be ignored.